### PR TITLE
feat: Apollo and GraphQL

### PR DESCRIPTION
**Pull request by Gabrielmong**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -3,11 +3,34 @@ import ReactDOM from 'react-dom/client'
...
 import App from './App'
 import reportWebVitals from './reportWebVitals'
+import {
+  ApolloClient,
+  InMemoryCache,
+  ApolloProvider,
+  gql
+} from '@apollo/client'
+
+const client = new ApolloClient({
```
Comment (carloshv93) [inline src/index.tsx:13]: Remove all code related to the console log

File: src/App.tsx [hunk 2]
```diff
@@ -63,6 +82,7 @@ function App() {
             changeStatus={changeStatus}
           />
         </div>
+        {renderGraphQL()}
```
Comment (carloshv93) [inline src/App.tsx:85]: This approach is like having a component within a component.

It is okay due to this is just temporally code, but not in a real code

File: src/App.tsx [hunk 3]
```diff
@@ -44,6 +45,24 @@ function App() {
...
 
+  const HELLO_WOLRD = gql`
+    query HelloWorld {
+      hello
+    }
+  `
+
+  const { loading, error, data } = useQuery(HELLO_WOLRD)
+
+  function renderGraphQL () {
```
Comment (carloshv93) [inline src/App.tsx:56]: This seems to be a completely different componennt

File: src/App.tsx [hunk 4]
```diff
@@ -44,6 +45,24 @@ function App() {
     )
   }
 
+  const HELLO_WOLRD = gql`
```
Comment (carloshv93) [inline src/App.tsx:48]: For the following "real code", let's have a folder for GraphQL with two subfolders: queries and mutations

and store all the GraphQL code in just on folder

**Review by carloshv93 — APPROVED** [review]
Please check the comments, I know it is a temp code. but lets try to start building it as better as we can

**Comment by Gabrielmong** [comment]
@carloshv93 Made appropriate changes, Created a GraphQL folder with two subfolders, and the same hello world query in there, changed up the rendering of the HelloWorld

**Review by carloshv93 — APPROVED** [review]
Loving it

**Pull request merged**